schemas/outcomes.v1.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "outcome",
+  "type": "object",
+  "properties": {
+    "timestamp": {
+      "type": "string"
+    },
+    "org_id": {
+      "type": ["integer", "null"]
+    },
+    "project_id": {
+      "type": ["integer", "null"]
+    },
+    "key_id": {
+      "type": ["string", "null"]
+    },
+    "outcome": {
+      "type": "integer"
+    },
+    "reason": {
+      "type": ["string", "null"]
+    },
+    "event_id": {
+      "type": ["string", "null"]
+    },
+    "category": {
+      "type": ["integer", "null"]
+    },
+    "quantity": {
+      "type": ["integer", "null"]
+    }
+  },
+  "required": [
+    "timestamp",
+    "org_id",
+    "project_id",
+    "key_id",
+    "outcome",
+    "reason",
+    "event_id",
+    "category",
+    "quantity"
```

Comment:
Are they all required?

Reply:
I'm not sure, what can I do to verify what fields are required and which are not?